Sources/MockoloFramework/Templates/PropertyMockable.swift

```diff
@@ -1,121 +0,0 @@
-//
```

Comment:
This file is initially broken. `MockoloFramework` is not expected to be used from the test runtime.

There are no ways to use the implementations in this file.